admin/index.html

```diff
@@ -1,4 +1,5 @@
 ---
+---
```

Comment:
This file only had one line of `---`, but to make it frontmatter, it must be two lines of it. As a result, the rendered output file began with `---`, which screwed up the HTML parser and caused pa11y to fail. Fixing the frontmatter fixed everything else. By far the easiest of the three!